### DISCUSSION TITLE
Hosting

### DISCUSSION THREAD

**Discussion by ryands17** [logdyhq/logdy-core#25] [Q&A] [answered]
Would you recommend exposing Logdy from my VM over https and protect it behind some kind of auth?

**Comment by PeterOsinski** [comment]
Hi, you can already do that! Take a look at this [blog post](https://logdy.dev/blog/post/remote-host-logs-browser-ui) and this [option](https://logdy.dev/docs/reference/cli#webserver-ip)

You can basically setup Logdy with authentication in the UI (`--ui-pass` option) and choose IP to which you would like to bind it.

**Reply by ryands17** [reply, marked as answer]
Cannot see the option in the CLI page unfortunately:
![CleanShot 2024-05-17 at 21 18 58@2x](https://github.com/logdyhq/logdy-core/assets/19697099/d0da2354-2b4c-4125-bc4b-a8d5c369237a)

**Reply by PeterOsinski** [reply]
Hey sorry, I just found out some issues on the homepage. Looking into it now

**Reply by PeterOsinski** [reply]
It's working now! 🙏🏻
